Traceback (most recent call last):
  File "main.py", line 5, in <module>
    for i in range (int(N)):
ValueError: invalid literal for int() with base 10: 'kmnl'
